tests/parser/exceptions/test_constancy_exception.py

```diff
@@ -76,16 +76,111 @@ def foo():
     for i in range(x):
         pass""",
         """
-f:int128
```

Comment:
This test now throws for its other issue of call violation, which is checked first in `visit_Expr`. Hence, I removed it as there are already tests for call violation in this [file](https://github.com/vyperlang/vyper/blob/master/tests/parser/exceptions/test_call_violation.py).